server/neptune/gateway/pr/signaler.go

```diff
@@ -18,6 +18,7 @@ type signaler interface {
 const (
 	Deprecated = "deprecated"
 	Destroy    = "-destroy"
+	Manifest   = "manifest_path"
```

Comment:
Maybe add a comment here that this is lyft specific.

Reply:
oop missed this before merging, will add to another pr